camel-core/src/main/java/org/apache/camel/impl/verifier/DefaultComponentVerifier.java

```diff
@@ -123,7 +123,7 @@ protected void verifyParametersAgainstCatalog(ResultBuilder builder, Map<String,
             stream(result.getInvalidEnum())
                 .map(entry ->
                     ResultErrorBuilder.withIllegalOption(entry.getKey(), entry.getValue())
-                        .detail(ResultErrorBuilder.attribute("enum.values"), result.getEnumChoices(entry.getKey()))
+                        .detail("enum.values", result.getEnumChoices(entry.getKey()))
```

Comment:
enum.values can be part of StandardAttribute imho as it something you can get form standard validation against camel catalog so it is cross component and I do not think there is any issue with lower case enums (test it with mvn install -Psourcecheck) but I would keep it uppercase as it is visually clear that it is a constant/enum (at least for me). 

Then about the wire representation, I like the "std.x.y" style so if you have a way to tweak the json serialization stuffs go ahead :)